chia/_tests/environments/wallet.py

```diff
@@ -26,6 +28,22 @@
 from chia.wallet.wallet_node_api import WalletNodeAPI
 from chia.wallet.wallet_state_manager import WalletStateManager
 
+STANDARD_TX_ENDPOINT_ARGS: dict[str, Any] = dict(
+    rpc_info=NeedsWalletRPC(client_info=None, wallet_rpc_port=None, fingerprint=None),
+    tx_config_loader=NeedsTXConfig(
+        min_coin_amount=cli_amount_none,
+        max_coin_amount=cli_amount_none,
+        coins_to_exclude=(),
+        amounts_to_exclude=(),
+        reuse=None,
+    ),
+    transaction_writer=TransactionsOut(transaction_file_out=None),
+    fee=uint64(0),
+    push=True,
+    valid_at=None,
+    expires_at=None,
+)
```

Comment:
gets us type checking on the standard values anyways.  though i do wonder bigger picture about having a bit less `dict` and unpacking around this and bit more class instance and `dataclasses.replace()` potentially.  though i haven't thought through this for all the use cases.  if possible, that would expand the type checking further now that mypy is aware of `dataclasses.replace()`.

```suggestion
STANDARD_TX_ENDPOINT_ARGS: dict[str, Any] = dataclasses.asdict(
    TransactionEndpoint(
        rpc_info=NeedsWalletRPC(client_info=None, wallet_rpc_port=None, fingerprint=None),
        tx_config_loader=NeedsTXConfig(
            min_coin_amount=cli_amount_none,
            max_coin_amount=cli_amount_none,
            coins_to_exclude=(),
            amounts_to_exclude=(),
            reuse=None,
        ),
        transaction_writer=TransactionsOut(transaction_file_out=None),
        fee=uint64(0),
        push=True,
        valid_at=None,
        expires_at=None,
    ),
)
```

Reply:
Took this, but it required some extra modifications around `TransactionEndpoint` which I actually think was quite helpful because it made mypy catch more stuff